commit 807330b464a55be19d4d5117e5f6b161f00a2f88
Author: Loren Huang <b02279@freescale.com>
Date:   Fri May 9 22:29:42 2014 +0800

    ENGR00308233-2 [#791]Refine wclip workaround algorithm 001
    
    Vivante patch name:0001-more-refinements-for-wclip-issue
    
    Date: Apr 14, 2014
    
    Signed-off-by: Loren Huang <b02279@freescale.com>
    Acked-by: Shawn Guo

diff --git a/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_types.h b/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_types.h
index b446529..bf1bb13 100644
--- a/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_types.h
+++ b/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_types.h
@@ -18,7 +18,6 @@
 *
 *****************************************************************************/
 
-
 #ifndef __gc_hal_types_h_
 #define __gc_hal_types_h_
 
@@ -1048,7 +1047,16 @@ typedef enum _gcePATCH_ID
     gcePATCH_DUOKANTV,
     gcePATCH_TESTAPP,
     gcePATCH_GOOGLEEARTH,
-
+    gcePATCH_SF4,
+    gcePATCH_SPEEDRACE,
+    gcePATCH_AIRNAVY,
+    gcePATCH_F18NEW,
+    gcePATCH_F18,
+    gcePATCH_WISTONESG,
+    gcvPATCH_VECUNIT_RED,
+    gcvPATCH_NAMESGAS,
+    gcvPATCH_AFTERBURNER,
+    gcvPATCH_UIMARK,
     /* Count enum*/
     gcePATCH_COUNT,
 }
